src/widget/widget.cpp

```diff
@@ -1103,20 +1106,20 @@ bool Widget::isFriendWidgetCurActiveWidget(Friend* f)
 
 bool Widget::event(QEvent * e)
 {
-    switch(e->type()) {
+    switch(e->type())
+    {
         case QEvent::WindowActivate:
             if (activeChatroomWidget != nullptr)
             {
                 activeChatroomWidget->resetEventFlags();
                 activeChatroomWidget->updateStatusLight();
             }
-        case QEvent::MouseButtonPress:
-        case QEvent::MouseButtonRelease:
-        case QEvent::Wheel:
-        case QEvent::KeyPress:
-        case QEvent::KeyRelease:
-            if (autoAwayActive)
-                onUserAwayCheck();  // Just so we get back from away faster when interacting with app
```

Comment:
Its pretty much useless noise. It is triggered only when user get back from away and does any of those actions in app window. Basically starts typing to last selected contact. Window has to be in focus and right contact has to be open. Chance is real slim of that happening. Most of the time user will either start moving mouse and it will trigger away reset via timer. So i figured for the sake of cleaner code we can get rid of this without loosing anything.